extensions/README.md

```diff
@@ -127,3 +127,12 @@ An example of this can be seen in a Landsat example:
     "l8:image_quality_oli": 9
   },
 ```
+
+### Directory Structure
+
+A STAC extension can have references to additional schemas within the extension schema. An example of this is the STAC item spec and the accompyning geojson spec.
+
+In order to keep these files together to preserve relative `ref` links the files should be grouped.
```

Comment:
Is the `ref` meant to be `href`? And the sentence sounds a little strange overall. Is there somewhere a word missing?

Reply:
I've changed it to `$ref`as it's used in the json schema files and cleaned up the description a bit.